diff --git a/contrib/svt-av1/module.defs b/contrib/svt-av1/module.defs
index 7c9116ecf..7a2ddde79 100644
--- a/contrib/svt-av1/module.defs
+++ b/contrib/svt-av1/module.defs
@@ -1,9 +1,8 @@
 $(eval $(call import.MODULE.defs,SVT-AV1,svt-av1))
 $(eval $(call import.CONTRIB.defs,SVT-AV1))
 
-SVT-AV1.FETCH.url     = https://github.com/HandBrake/HandBrake-contribs/releases/download/contribs2/SVT-AV1-v4.2.0.tar.gz
-SVT-AV1.FETCH.url    += https://gitlab.com/AOMediaCodec/SVT-AV1/-/archive/v4.2.0/SVT-AV1-v4.2.0.tar.gz
-SVT-AV1.FETCH.sha256  = c7b13c4a84bd3751aa35fcc72be13e6875467e7c2216879251a486e5b1e4e740
+SVT-AV1.FETCH.url     = https://github.com/nekotrix/HandBrake-SVT-AV1-Essential/releases/download/contribs/SVT-AV1-Essential-4.0.1-Essential.tar.gz
+SVT-AV1.FETCH.sha256  = 4575b5b50eb6888f358d235e44c6caf1a968812ce2200692b2025e5ea082244b
 
 SVT-AV1.GCC.args.c_std =
 
