diff --git a/contrib/svt-av1/module.defs b/contrib/svt-av1/module.defs
index bfc198a57..6681ee92b 100644
--- a/contrib/svt-av1/module.defs
+++ b/contrib/svt-av1/module.defs
@@ -1,9 +1,8 @@
 $(eval $(call import.MODULE.defs,SVT-AV1,svt-av1))
 $(eval $(call import.CONTRIB.defs,SVT-AV1))
 
-SVT-AV1.FETCH.url     = https://github.com/HandBrake/HandBrake-contribs/releases/download/contribs2/SVT-AV1-v3.1.0.tar.gz
-SVT-AV1.FETCH.url    += https://gitlab.com/AOMediaCodec/SVT-AV1/-/archive/v3.1.0/SVT-AV1-v3.1.0.tar.gz
-SVT-AV1.FETCH.sha256  = 3999586c261dc3d8690fd1489fc74da4e0fdff9159c8ce2b76ddfac001ad96d3
+SVT-AV1.FETCH.url     = https://github.com/nekotrix/HandBrake-SVT-AV1-Essential/releases/download/contrib/SVT-AV1-Essential.tar.gz
+SVT-AV1.FETCH.sha256  = 8c46707d41127a992c60d82249a737fac8061076650c19fdc559436e1edd8b4a
 
 SVT-AV1.GCC.args.c_std =
 
